ent/proto/entpb/generate.go

```diff
@@ -1,4 +1,4 @@
 package entpb
 
-//go:generate protoc -I=.. --go_out=.. --go-grpc_out=.. --go_opt=paths=source_relative --go-grpc_opt=paths=source_relative --entgrpc_out=.. --entgrpc_opt=paths=source_relative,schema_path=../../schema entpb/entpb.proto
+//go:generate protoc -I=.. --go_out=.. --go-grpc_out=.. --go_opt=paths=source_relative --go-grpc_opt=paths=source_relative --entgrpc_out=.. --entgrpc_opt=paths=source_relative,schema_path=../../schema entpb/entpb.proto entpb/ext.proto
```

Comment:
This does not work here. I basically get

"--entgrpc_out: entproto: type "Ext" of service "ExtService" not found in graph"

which is triggered by line 178 of main.go within entproto.

Reply:
@rotemtam I get the same error. Is this functionality still being worked on?

Reply:
Apologies, missed this thread . Will reply soon 🙏

Reply:
Please upgrade to the most recent version of `protoc-gen-entgrpc`:
```
 go install entgo.io/contrib/entproto/cmd/protoc-gen-entgrpc@master
```
It contains [this commit](https://github.com/ent/contrib/commit/54eb1286ff700e75cd0883827e583e2952dd699e) from February that addresses the issue. 

I will update the tutorial to reflect this.

Apologize for the confusion